v2/component/public/bundled/services/Data.js

```diff
@@ -54,13 +54,11 @@ Vue.component("data-mixin", {
 			paging: {},
 			state: {},
 			filter: {},
-			loadTimer: null
+			loadTimer: null,
+			isLoading: false,
```

Comment:
Maybe `isLoading` is too generic as this this property will be added to the component where the mixin is used.